test/login.tests.js

```diff
@@ -563,10 +563,9 @@ describe('auth', () => {
     });
     assert.equal(res.statusCode, 500);
     console.log(res.body.err.message);
-    assert.match(
+    assert.equal(
       res.body.err.message,
-      /^Issuer.discover\(\) failed/,
-      'Should get error json from server error middleware'
```

Comment:
Error message returned by the new version is not very nice here, but it is what it is ;-)